tests/unit/test_svs.cpp

```diff
@@ -2553,6 +2554,128 @@ TEST(SVSTest, quant_modes) {
     }
 }
 
+TEST(SVSTest, save_load) {
+    namespace fs = std::filesystem;
+    // Limit VecSim log level to avoid printing too much information
+    VecSimIndexInterface::setLogCallbackFunction(svsTestLogCallBackNoDebug);
+
+    const size_t dim = 4;
+    const size_t n = 100;
+    const size_t k = 10;
+
+    for (auto quant_bits : {VecSimSvsQuant_NONE, VecSimSvsQuant_Scalar, VecSimSvsQuant_8,
+                            VecSimSvsQuant_4, VecSimSvsQuant_4x4, VecSimSvsQuant_4x8,
+                            VecSimSvsQuant_4x8_LeanVec, VecSimSvsQuant_8x8_LeanVec}) {
+        SVSParams params = {
+            .type = VecSimType_FLOAT32,
+            .dim = dim,
+            .metric = VecSimMetric_L2,
+            .blockSize = 1024,
+            /* SVS-Vamana specifics */
+            .quantBits = quant_bits,
+            .graph_max_degree = 63, // x^2-1 to round the graph block size
+            .construction_window_size = 20,
+            .max_candidate_pool_size = 1024,
+            .prune_to = 60,
+            .use_search_history = VecSimOption_ENABLE,
+        };
```

Comment:
Should we test the case where index parameters are zeroed and ensure the loaded index uses default parameters?

Reply:
Does SVSParams support default? 
I don't see it

Reply:
see svs_details::getOrDefault calls